src/_shared/js/capi-images.js

```diff
@@ -46,10 +46,10 @@ function createPicture (imageInfo, imageElem) {
 }
 
 // Inserts an image into the card, using the data derived from cAPI.
-export function insertImage (imageContainer, imageInfo, override) {
+export function insertImage (imageContainer, imageInfo, classes, override) {
 
 	let image = document.createElement('img');
-	image.classList.add('advert__image');
+	image.classList.add(...classes);
```

Comment:
At this point, it is probably less expensive to just assign `image.className = classes.join(' ')`